### PR TITLE
doc: add contributing doc

### LINKED ISSUE #15: Add contributing guidelines
Create a CONTRIBUTING.md for the repository.

### PR DESCRIPTION
**Pull request by exdx**
Closes #15

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+Rukpak is an Apache 2.0 licensed project and accepts contributions via GitHub pull requests (PRs).
+
+This document outlines some conventions on commit message formatting, contact points for developers, and other resources
+to help RukPak contributors.
+
+## Communication
+
+- Email: [operator-framework-olm-dev](operator-framework-olm-dev@googlegroups.com)
+- Slack: [#olm-dev][olm-dev]
+- Working Group: [olm-wg][olm-wg]
```
Comment (timflannagan) [inline CONTRIBUTING.md:12]: nit: broken links?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+Proposing larger changes to the RukPak project may require an enhancement proposal, or some documentation, before being
+considered. Any change to RukPak's existing behavior or features, APIs, or changes and additions to tests do not require
+an enhancement proposal.
+
+## Contribution flow
+
+This is a rough outline of what a contributor's workflow looks like:
+
+- Identify or create an issue.
+- Create a topic branch from where to base the contribution. This is usually the master branch.
```
Comment (timflannagan) [inline CONTRIBUTING.md:37]: s/master/main

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+### Code style
+
+The coding style suggested by the Golang community is used throughout the RukPak project:
+
+- [CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments)
+- [EffectiveGo](https://golang.org/doc/effective_go)
+
+In addition to the linked style documentation, OLM formats Golang packages using the `gofmt`
+and `goimports` tooling. Before submitting a PR, please run `make lint` locally and commit the results. This will help
+expedite the review process, focusing less on style conflicts, and more on the design and implementation details.
```
Comment (timflannagan) [inline CONTRIBUTING.md:71]: We need to fix this in OLM's contributing documentation, but we use whatever we'd defined as linters in our goreleaser configuration.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+Please follow this style to make the RukPak project easier to review, maintain and develop.
+
+## Documentation
```
[the remark this answers is not in the document]
Comment (exdx) [reply CONTRIBUTING.md:75]: Good catch

**Pull request merged**